try:
    i = int(input("enter the number : "))

except Exception as e:
    print(e)

else:
    print("if try block run sucesses fuly then else will exicute")